src/SprykerEco/Yves/Braintree/Theme/default/components/molecules/braintree-payment-form/braintree-payment-form.ts

```diff
@@ -1,21 +1,34 @@
 import Component from 'ShopUi/models/component';
 import ScriptLoader from 'ShopUi/components/molecules/script-loader/script-loader';
-import dropin from 'braintree-web-drop-in';
 
-interface braintreeConfig {
-    nonce: string,
-    type: string
+interface BraintreeConfig {
+    nonce: string;
+    type: string;
 }
 
-interface braintreeErrorConfig {
-    message: string
+interface BraintreeErrorConfig {
+    message: string;
+}
+
+interface PaymentMethodHandler {
+    (response: BraintreeConfig): void;
+}
+
+interface ErrorHandler {
+    (error: BraintreeErrorConfig): void;
+}
+
+interface BraintreeSetupSettings {
+    onPaymentMethodReceived: PaymentMethodHandler;
+    onError: ErrorHandler;
```

Comment:
Why do we need additional interfaces for it? We can use they as values

```
interface BraintreeSetupSettings {
    onPaymentMethodReceived: (response: BraintreeConfig) => void;;
    onError: (error: BraintreeErrorConfig) => void;;
}
```

Reply:
Using a property signature of function type is disable by `prefer-method-signature` rule of the tslinter

Reply:
Refactored with 
`interface BraintreeSetupSettings {
    onPaymentMethodReceived(response: BraintreeConfig): void;
    onError(error: BraintreeErrorConfig): void;
}`